src/gui/screen_menu_steel_sheets.cpp

```diff
@@ -0,0 +1,177 @@
+    }
+};
+
+using SheetProfileMenuScreen = ScreenMenu<EHeader::Off, EFooter::On, HelpLines_None, MI_RETURN, MI_SHEET_SELECT, MI_SHEET_CALIBRATE,
+    MI_SHEET_RENAME, MI_SHEET_RESET>;
+
+template <typename Index>
+class SheetProfileMenuScreenT : public SheetProfileMenuScreen {
+public:
+    constexpr static const char *label = N_("Sheet Profile");
+    using index_type = Index;
+    SheetProfileMenuScreenT()
+        : SheetProfileMenuScreen(_(label)) {
+        if (sheet_is_calibrated(Index::value)) {
+            Item<MI_SHEET_SELECT>().Enable();
+            Item<MI_SHEET_RESET>().Enable();
+        }
+        if (Index::value == 0)
+            Item<MI_SHEET_RESET>().Disable();
+    }
+
+    virtual void windowEvent(EventLock /*has private ctor*/, window_t *sender, GUI_event_t ev, void *param) override {
+        SERIAL_ECHOLN("SheetProfile::event");
+        if (ev != GUI_event_t::CHILD_CLICK) {
+            SuperWindowEvent(sender, ev, param);
+            return;
+        }
+        profile_action action = static_cast<profile_action>((uint32_t)param);
+        switch (action) {
+        case profile_action::Reset:
+            if (sheet_reset(Index::value)) {
+                Item<MI_SHEET_RESET>().Disable();
+                Item<MI_SHEET_SELECT>().Disable();
+                SERIAL_ECHOLN("MI_SHEET_RESET OK");
+            } else
+                SERIAL_ECHOLN("MI_SHEET_RESET FAIL!");
+            break;
+        case profile_action::Select:
+            SERIAL_ECHOLN("MI_SHEET_SELECT");
+            sheet_select(Index::value);
+            break;
+        case profile_action::Calibrate:
+            SERIAL_ECHOLN("MI_SHEET_CALIBRATE");
+            sheet_calibrate(Index::value);
+            ScreenWizard::RunFirstLayerStandAlone();
+            break;
+        case profile_action::Rename:
+            SERIAL_ECHOLN("MI_SHEET_RENAME");
+            Screens::Access()->Open([]() {
+                screen_sheet_rename_t::index(Index::value);
+                return ScreenFactory::Screen<screen_sheet_rename_t>();
+            });
+            break;
+        default:
+            SERIAL_ECHOPAIR("Click: ", static_cast<uint32_t>(action));
+            SERIAL_ECHOLN("");
+            break;
+        }
+    }
+
+private:
+};
+
+template <typename Index>
+struct profile_record_t : public WI_LABEL_t {
+    profile_record_t()
+        : WI_LABEL_t(nullptr, 0, true, false) {
+        char name[MAX_SHEET_NAME_LENGTH];
+        memset(name, 0, MAX_SHEET_NAME_LENGTH);
+        sheet_name(Index::value, name, MAX_SHEET_NAME_LENGTH);
+        SetLabel(name);
+    };
+
+protected:
+    virtual void click(IWindowMenu &window_menu) override {
+        Screens::Access()->Open([]() {
+            return ScreenFactory::Screen<SheetProfileMenuScreenT<Index>>();
+        });
+    }
+};
+
+using Screen = ScreenMenu<EHeader::Off, EFooter::On, HelpLines_None, MI_RETURN
+#if (EEPROM_FEATURES & EEPROM_FEATURE_SHEETS)
```

Comment:
I think `#if (EEPROM_FEATURES & EEPROM_FEATURE_SHEETS)` should exclude entire file (disable entire sheet menu)